src/cloud/social/alias/randombytes.js

```diff
@@ -1,33 +1,55 @@
 // Alternative to crypto-browserify's randombytes shim which
```

Comment:
Let's keep a reference to what we're shimming, i.e.:
https://github.com/crypto-browserify/randombytes/blob/master/browser.js